node/src/actor_system_factory.rs

```diff
@@ -982,24 +1014,24 @@ mod tests {
                 ),
             },
         };
-        let persistent_config =
+        let mut persistent_config =
             PersistentConfigurationMock::default().chain_name_result("eth-ropsten".to_string());
         Bootstrapper::pub_initialize_cryptdes_for_testing(
             &Some(main_cryptde()),
             &Some(alias_cryptde()),
         );
         let mut tools = ActorSystemFactoryToolsReal::new();
         tools.automap_control_factory = Box::new(
-            AutomapControlFactoryMock::new().make_result(
+            AutomapControlFactoryMock::new().make_result(Box::new(
```

Comment:
Maybe this doesn't need to be Boxed? I dunno--you seemed to want me to mark this.

Reply:
Nope, I cannot. A bit of disappointment here.